_includes/data/access.html

```diff
@@ -1,47 +1,99 @@
     <!-- Data Section -->
-        <div class="container">
-       		<div class="col-lg-11 col-lg-offset-0">
+<script type="text/javascript">
+		function sendMail( subject, body, cc='', bcc='') {
+			// ATTENTION! This mail address might be scraped from the page for spam.
+			let recepientMail = "temp@robot-mail.com"; //getnada dot com
 
-		        <h2>Request Access</h2>
-		        <form name="accessForm" action="" method="POST">
+			let mailLink = "mailto:" + recepientMail +
+								"?cc=" + cc + 
+								"&bcc=" + bcc + 
+								"&subject=" + subject +
+								"&body=" + body;
+			let encodedMailLink = encodeURI(mailLink);
+			window.open(encodedMailLink, "_blank");
+		}
+
+		function requestAccess(accessForm) {
+			
+			let cc = '';
+			let bcc = '';
+			let subject = '';
+			let body = '';
+
+			let formElements = accessForm.elements;
+
+			let submitDate = Date.now(), 
+				applicant = formElements.namedItem('applicant').value,
+				applicantPosition = formElements.namedItem('applicantPosition').value,
+				principalInvestigator = formElements.namedItem('principalInvestigator').value,
+				institution = formElements.namedItem('institution').value,
+				datasets = formElements.namedItem('datasets').value,
+				projectOutline = formElements.namedItem('projectOutline').value,
+				hypothesis = formElements.namedItem('hypothesis').value,
+				ethicalVote = formElements.namedItem('ethicalYes').checked;
+
+			body = `To Whom it May Concern,
+
+I am ${applicant} and I work as ${applicantPosition} at ${institution}.
+The principal investigator is ${principalInvestigator}.
+
+We sould like to get access to ${datasets} and plan to use the data in the for the following project:
+${projectOutline}
 
+Our hypothesis is that ${hypothesis}
```

Comment:
Not sure, whether the insertion like this will result in a pleasing outcome. I guess the user will write pretty stand-alone texts. So maybe better
```suggestion
I want to test the following hypothesis:

${hypothesis}
```